packages/angular-material/README.md

```diff
@@ -0,0 +1,22 @@
+# JSONForms - More Forms. Less Code
+### Complex Forms in the blink of an eye
+
+JSONForms uses HTML custom elements and eliminates the need to write HTML templates in order to create forms by leveraging the capabilities of JSON and JSON schema.
```

Comment:
This sentence actually isn't entirely true anymore since we only use custom elements in the context of the webcomponent package. Maybe we should update the wording here (this need to be updated across all READMEs, I think).

Reply:
Maybe something along the line of: JSONForms eliminates the tedious task of writing fully-featued forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript? @koegel Your take on this?